geoapps_utils/driver/params.py

```diff
@@ -92,9 +93,11 @@ def _initialize(self, **kwargs):
                 validations=self.validations,
                 validate=False,
             )
-        self.update(self.input_file.data)
+        input_file = self.input_file
+        assert input_file is not None
```

Comment:
Same here

Reply:
this is for sure an invariant: cannot be null by construction.
The code above call the property setter with a newly created input file if it was null.
But we still have to state that the property getter return a non-null value